pkg/controller/buildrun/buildrun_controller.go

```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 
 	buildv1alpha1 "github.com/redhat-developer/build/pkg/apis/build/v1alpha1"
-	taskv1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	v1beta1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
```

Comment:
Using `v1beta1` as import alias implies updating it when the version changes, so I suggest using a name like `tekton`, which is going to be the same for future versions.

Reply:
I see, nice point. Let me change this.

Reply:
@otaviof one comment, we still have the `github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1` package around, is not possible to fully remove it because in the tekton v1beta1 they still have references to it. For example, see this line https://github.com/tektoncd/pipeline/blob/master/pkg/apis/pipeline/v1beta1/resource_types.go#L103 . This goes from the v0.11.0 till their develop branch. So maybe we should stick for a little while with the distinctions of `v1beta1` and `taskv1` alias for more time. 

Also, in this PR if you look at https://github.com/redhat-developer/build/pull/180/files#diff-c8bae8bebf9ceb3609b947afd6c4af05R10 , you will see that the `generate_taskrun.go` class, import both packages versions.

Reply:
@otaviof pls resolve if you agree on this, so we can get this merge soon.